src/topics.ts

```diff
@@ -33,3 +33,15 @@ export function compileTopics(
 function hexToBytes(input: string): Uint8Array {
   return Buffer.from(input.substring(2), 'hex');
 }
+
+export function toStringArray(input: string): Array<string>{
+  return input
+  .split(',')
+  .map(
+    (t: string) => 
+    t.replace("{\"\\\\", '0')
+    .replace("\"\\\\", "0")
+    .replace("\"}", "")
+    .replace("\"", "")
```

Comment:
This seems pretty fragile, and I'm not clear on why it's needed. Could you elaborate a bit, @0x3bfc?

Reply:
Totally agree! 
Will try to find a way to convert the buffer array from postgress to an array of hex values.